### ISSUE TITLE
When getting item images, item image names must be all caps to return

### ISSUE THREAD

**Issue by BrianRoyal** [NCRCounterpointAPI/APIGuide#29]
Example:
When I run a GET on https://[myserver]/Item/GTEST3/Images/, I can successfully get results, but only if the image names in windows on the CounterPoint server are all caps.

If my image name is gtest3.jpg or GTest3.jpg or GTESt3.jpg, I get the following result:
```
{
    "ErrorCode": "ERROR_RECORD_NOT_FOUND",
    "Message": "No item images found for item GTEST3"
}
```

However, if I change the filename to GTEST3.jpg, I get
```
{
    "IMAGE_FILENAMES": [
        "GTEST3.jpg"
    ],
    "ErrorCode": "SUCCESS"
}
```

To further complicate matters, the name must be GTEST3.jpg, with a fully capitalized file name but a lowercase extension (.jpg).  If I use an uppercase extension (GTEST.JPG), I get
```
{
    "ErrorCode": "ERROR_RECORD_NOT_FOUND",
    "Message": "No item images found for item GTEST3"
}
```

**Comment by MikeRitchie** [comment]
This seems like it's likely an easy fix, so I'll look to see if we can include it in our existing API 1.1 release scope (targeting late August). In the meantime, it sounds like you have a workaround available of naming the filenames properly.

**Comment by MikeRitchie** [comment]
This fix will be included in the upcoming 1.1 release
